src/OctoshiftCLI.Tests/Integration/GenerateArchiveTests.cs

```diff
@@ -0,0 +1,28 @@
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace OctoshiftCLI.Tests.Integration
+{
+    public class GenerateArchiveTests
+    {
+        #region Script generation and verification helpers
+        private async Task<string> GenerateArchive(string scenarioName, string additionalFlags)
+        {
+            var outputFilename = $"{System.IO.Path.GetTempPath()}{scenarioName}.sh";
+            File.Delete(outputFilename);
+            var parameterString = $"generate-archive --github-source-org GuacamoleResearch --github-url https://mygithub.com/api --github-source-repo myrepo";
+            var parameters = parameterString.Trim().Split(' ');
+
+            await OctoshiftCLI.GithubEnterpriseImporter.Program.Main(parameters);
+            return "";
+        }
+        #endregion
+
+        [Fact]
+        public async void Should_Fail_To_Generate_With_Invalid_Parameters()
+        {
+            var response = await GenerateArchive("invalid-parameters", "--unsupport-parameter");
```

Comment:
I don't think there should be an INT test for this command, it can easily be converted to a unit test. We tend to write scenario based INT tests but this is just testing the `generate-archive` command.

Reply:
Yeah, I'm about to delete all the existing integration test stuff in another PR and commit a complete rewrite of the integration tests. For this PR unit tests should suffice.